### DISCUSSION TITLE
Support for more examples of Console App

### DISCUSSION THREAD

**Discussion by iio888** [elsa-workflows/elsa-core#5166] [Ideas]
It can be easily noticed that most of documents and samples of Elsa are based on Web Application. Would it be better to add more documents or examples for Console App?
Such as usages of `WorkflowRuntime` `Alterations`.

Thanks a lot !

**Comment by glime-ai** [comment]
It seems you are looking for more documentation and examples on how to use Elsa Workflows within a console application context, specifically regarding `WorkflowRuntime` and `Alterations`. While the majority of the documentation and samples may focus on web applications, Elsa is designed to be flexible and can be used in various application types, including console applications.

The Elsa documentation does provide a guide on setting up Elsa in a console application, as seen in the source from `https://v3.elsaworkflows.io/docs/installation/elsa-console`. This guide walks you through creating a new console application, installing the Elsa NuGet package, and running a simple workflow.

For more complex scenarios, such as using `WorkflowRuntime` and `Alterations`, you might need to delve deeper into the API and possibly look at the source code for examples of how these are used. The Elsa GitHub repository could be a good place to start, as it may contain examples and discussions that are not covered in the official documentation.

If you feel that the documentation is lacking in this area, you could consider opening an issue on the Elsa GitHub repository to request more examples and documentation for console applications. The maintainers and community might then provide additional resources or point you to existing materials that can help.

Would you like to proceed with more specific questions about `WorkflowRuntime` and `Alterations`, or do you need guidance on how to request additional documentation from the Elsa community?
